puppet/zulip_ops/manifests/profile/prometheus_server.pp

```diff
@@ -0,0 +1,61 @@
+# @summary Gathers Prometheus statistics from all nodes.  Only one
+# instance is necessary.
+#
+class zulip_ops::profile::prometheus_server {
+  include zulip_ops::profile::base
+  include zulip_ops::prometheus_node
+
+  $version = '2.27.1'
+  zulip::sha256_tarball_to { 'prometheus':
+    url     => "https://github.com/prometheus/prometheus/releases/download/v${version}/prometheus-${version}.linux-amd64.tar.gz",
+    sha256  => 'ce637d0167d5e6d2561f3bd37e1c58fe8601e13e4e1ea745653c068f6e1317ae',
+    install => {
+      "prometheus-${version}.linux-amd64/" => "/opt/prometheus-${version}/",
```

Comment:
Do we want this under /srv/ for consistency with the other things we install?

Reply:
> I saw this had been updated recently, and figured now was a good time to do a skim; haven't tried to understand whether these are the right configuration options.
> 
> (One thing I wonder about is whether we're only setting 12 options in the grafana configuration, and we might be happier not checking in their template and instead just having a file that sets those 12 options).

Yeah, I was waffling on that.  I think it probably makes sense to do that.